### PR TITLE
Add osql snapinfo hash for currently running osql session

### PR DESCRIPTION
**Pull request by adizaimi**
Add osql snapinfo hash for currentnly running osql session

### CODE REVIEW

File: db/comdb2.h [hunk 1]
```diff
@@ -1194,7 +1194,11 @@ typedef struct snap_uid {
     char key[MAX_SNAP_KEY_LEN]; /* cnonce */
 } snap_uid_t;
 
-enum { SNAP_UID_LENGTH = 8 + 4 + (4 * 5) + 4 + 64 };
+enum { SNAP_UID_LENGTH = 16 + 4 + (4 * 5) + 4 + 64 };
+
+BB_COMPILE_TIME_ASSERT(snap_uid_size,
+                       sizeof(snap_uid_t) == SNAP_UID_LENGTH);
+
```
Comment (markhannum) [inline db/comdb2.h]: wait, this breaks compatibility, no?

Comment (markhannum) [reply db/comdb2.h]: !! uid mode is disabled by legacy defaults .. nice ..

**Comment by adizaimi** [comment]
/skipsysbench

trying roborivers one more time

File: db/comdb2.h (hunk 1 above)
Comment (adizaimi) [reply db/comdb2.h]: Just to clarify for the record, SNAP_UID_LENGTH was set to the wrong value and here I am just fixing it and adding the compile time assert.

**Pull request merged**